multinet/multinet/multinet.py

```diff
@@ -52,7 +55,13 @@ def _graphql():
     app.logger.debug("request: %s" % query)
     app.logger.debug("variables: %s" % variables)
 
-    result = graphql_query(query, variables)
+    try:
+        result = graphql_query(query, variables)
+    except ValueError as e:
+        for arg in e.args:
+            app.logger.error(arg)
+        abort(500)
```

Comment:
I don't think this is quite right: if the query succeeds in executing, but encounters problems, it's likely to be an issue with the input, and this should probably be a 400 rather than a 500.

Reply:
Can you change this to a 400?

Also, instead of using `abort()` I think it's better to use the ordinary return convention that Flask supports. That would be something like `return (result, 400)`. Since the next line returns a successful response, you could combine them like this:

```python
status_code = 400 if result["errors"] else 200
return (result, status_code)
```

Reply:
Also, since this location isn't where the DB connection issue is going to be detected, we may just want to revert the changes to this file altogether, or handle them in a separate PR.

Reply:
Opened #127